static/img/flags/de.svg

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
We should plan to optimize these svgs using `svgo` https://github.com/ajstarks/svgo, but we can always do that later :)